src/python/epicscorelibs/ioc.py

```diff
@@ -63,7 +64,7 @@ def ioc(cmd):
     '''
     return iocshCmd(cmd.encode())
 
-def start_ioc(database=None, macros='', dbs=None):
+def start_ioc(extra_dbd_load, extra_dso_load, database=None, macros='', dbs=None):
```

Comment:
New arguments should be appended, with a default which preserves the current behavior.